handbook/product/design/artifacts/index.md

```diff
@@ -0,0 +1,19 @@
+# Design artifacts
+
+In Sourcegraph’s globally distributed remote environment, design artifacts are crucial for sharing and collaborating on design—and a key part of how we [design in the open](../index.md#designing-in-the-open).
+
+We use Figma files, RFCs, Google Docs, GitHub issues, and more as artifacts. Artifacts give us a way to capture our ideas and thoughts, and then give us time to go away and give thoughtful consideration before providing feedback.
+
+Decisions are made in design artifacts. While we can talk about things in synchronous sessions like Slack conversations or video calls, by making decisions in artifacts, it helps others to understand the context around the decision and provide further feedback or ask questions in a self-contained space.
```

Comment:
```suggestion
Decisions are made in design artifacts. While we can talk about things in synchronous sessions like Slack conversations or video calls, making decisions in artifacts helps others to understand the context around the decision and provide further feedback or ask questions in a self-contained space.

Design artifacts should be treated as a source of truth, it applies in particular to Figma files. Designers are responsible for communicating this to their product teams and keeping files up-to-date. 
```

Reply:
@quinnkeast I'm trying to convey here that no matter what's discussed on slack / GitHub etc. Figma files are the source of truth. I don't think it applies as much to other artifacts (ex. RFC don't have to be up-to-date later in the design process)

Reply:
Makes sense! How about this slight reword so that it's clear that Figma files might not be the only type of artifact that ends up being the source of truth, but then when they are, it's our role to communicate + keep it updated?

```suggestion
Decisions are made in design artifacts. While we can talk about things in synchronous sessions like Slack conversations or video calls, making decisions in artifacts helps others to understand the context around the decision and provide further feedback or ask questions in a self-contained space.

When a design artifact like a Figma file is a source of truth, designers are responsible for communicating this to their teams and for keeping those artifacts up to date.
```